packages/website/docusaurus.config.js

```diff
@@ -29,20 +29,23 @@ const config = {
     '@docusaurus/plugin-debug',
     [
       '@docusaurus/theme-classic',
```

Comment:
I'm simply curious: why aren't you using presets anyways? You are virtually using all the plugins in the preset. Also, you can take advantage of shorthands if you move these to `themes` instead of putting them in `plugins`

Reply:
because we are using 2 docs, and initially i had issues when i attempted to use preset + doc plugin

Reply:
Hmm, I have used multi-instance with no problem... "missing id: 'rules-docs' on doc when used from preset" Do you mean the plugin ID, or the doc ID? You should be able to configure plugin IDs from preset options

Reply:
plugin id, as you can't have multiple instances of the same plugin without explicitly defining id